spec/controllers/idv/in_person/public/address_search_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Idv::InPerson::Public::AddressSearchController do
+  include Rails.application.routes.url_helpers
+
+  describe '#index' do
+    subject(:action) do
+      post :index,
+           params:
+            { address: '100 main' }
```

Comment:
this looks short enough to put on one line if we want
```suggestion
      post :index, params: { address: '100 main' }
```